eng/pipelines/templates/jobs/installation.check.yml

```diff
@@ -0,0 +1,33 @@
+parameters:
+  - name: Artifact
+    type: object
+    default: {}
+  - name: ArtifactName
+    type: string
+    default: "not-specified"
+
+jobs: 
+  - job: InstallationCheck
+    displayName: "Installation Check"
+    variables:
+      - template: ../variables/globals.yml
+    steps:
+      - download: current
+        artifact: ${{parameters.ArtifactName}}-signed
+      - pwsh: |
+          Write-Host "Copy-Item $(Pipeline.Workspace)/${{parameters.ArtifactName}}-signed/${{parameters.Artifact.groupId}}/${{parameters.Artifact.name}}/" "$(Pipeline.Workspace)/install-check-dir/${{parameters.Artifact.name}} -Recurse"
+          Copy-Item "${{parameters.ArtifactName}}-signed/${{parameters.Artifact.groupId}}/${{parameters.Artifact.name}}/" "install-check-dir/${{parameters.Artifact.name}}" -Recurse
```

Comment:
Why copy? Can't we just do the install from the location it is at?

Reply:
Ben suggested making a directory for it. I think it's safer to test in its own directory as well. Making sure we're not messing with the artifact we're about to ship

Reply:
I'm not opposed to it but we aren't going be shipping this copy of the artifact it will be the publish job download that will end up being published.